packages/design-system-web/src/components/geov-map-circles-popup/geov-map-circles-popup.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import { h } from '@stencil/core';
+import { stencilWrapper } from '../../../.storybook/lib/stencilWrapper';
+import { docsTemlpate } from '../../../.storybook/templates/docsTemplate';
+import componentApi from './docs-component-api.md?raw';
+import overview from './docs-overview.md?raw';
+export default {
+  title: 'Data Visualization Components/Map Circles/Popup',
+  tags: ['autodocs'],
+  parameters: {
+    viewMode: 'docs',
+    docs: {
+      page: () => docsTemlpate(overview, componentApi),
+    },
+  },
+};
+
+export const Default = await stencilWrapper(
+  <geov-map-circles-popup
+    style={{ height: '200px' }}
```

Comment:
this might be unnecessary..? doesn't change anything for me at least.

Reply:
I agree